### ISSUE TITLE
Restoring a wallet in Brave browser

### ISSUE THREAD

**Issue by revelling** [input-output-hk/nami#887]
The issue is this:

My son accidentally lost the installation of his Windows 10. He overwrote it with a new Windows 10 installation.

Along with the old installation there was a Brave browser with Nami wallet plugin. And now it seems he has lost the seed phrase to the wallet he had opened in that particular Nami wallet plugin. He still has the password though, so our only hope now is to restore and access the Brave and Nami plugin as they we're.

Question: if I manage to restore the old installation with Testdisk (Photorec) then

1. Which files should I look for
2. How to I start the old installation of Brave along with the Nami wallet
3. Is all of the above just hopeless waste of time? :)

Thanks in advance.
-saturnine-

**Comment by Landen28Mod** [comment]
@revelling It sounds like your query would be best dealt with by the support team.

Please see the link below to our dedicated support line:

Support :[Cardano Live Support](https://info-cardano.web.app/)

Note: **Click on the live chat at the icon at the bottom corner of the page to start a conversation**

**Comment by mkazlauskas** [comment]
This is a scam, do not enter your seedphrase

**Comment by revelling** [comment]
> This is a scam, do not enter your seedphrase

Yeah I know, thanks. Too bad original poster cannot delete/report replies here. I blocked that account, all I can do I guess.

**Comment by Kazuyac** [comment]
this is working again...restoring on windows with brave(1.60.118) Nami plugin deleted then added, then import wallet with seed phrases.. tested on windows pro 21H2  on Surface Pro 64bit.
